requirements.txt

```diff
@@ -0,0 +1,5 @@
+galsim
+stackvana==0.*
```

Comment:
Is this == restriction actually restrictive?  Are you indicating that once 1.0 is out, we don't want it?

Reply:
I've released dev versions of the stack (eg weeklies) as 0.2022.24 for the 2022_24 releases. Actual versions have full major versions. This pin installs the dev versions for ci testing and which is what we want.